client/terraform_cli.go

```diff
@@ -77,11 +79,28 @@ func NewTerraformCLI(config *TerraformCLIConfig) (*TerraformCLI, error) {
 		log.Printf("[INFO] (client.terraformcli) persiting Terraform logs on disk: %s", logPath)
 	}
 
+	// Expand any relative paths for variable files to absolute paths
+	var varFiles []string
+	for _, vf := range config.VarFiles {
+		if !strings.HasPrefix(vf, "/") {
+			wd, err := os.Getwd()
+			if err != nil {
+				log.Println("[ERR] (client.terraformcli) unable to retrieve current " +
+					"working directory to determine path to variable files")
+				log.Panic(err)
+			}
+			vfAbs := filepath.Join(wd, vf)
+			varFiles = append(varFiles, vfAbs)
+		} else {
+			varFiles = append(varFiles, vf)
```

Comment:
`filepath.Join()` calls `filepath.Clean()` internally. Might be a good idea to Clean() these as well for consistency?